scripts/zkai_attention_kv_d16_quantized_softmax_receipt_gate.py

```
@@ -0,0 +1,588 @@
+    add("source_input_denominator_zero", lambda _r, s, e: (s["score_rows"][0].__setitem__("weight_denominator", 0), e.__setitem__("source_input", s)))
+    add("source_input_weighted_value_drift", lambda _r, s, e: (s["score_rows"][0]["weighted_value"].__setitem__(0, 81), e.__setitem__("source_input", s)))
+    add("source_input_weighted_numerator_drift", lambda _r, s, e: (s["score_rows"][0]["weighted_numerator"].__setitem__(0, 561), e.__setitem__("source_input", s)))
+    add("source_input_remainder_drift", lambda _r, s, e: (s["score_rows"][0]["output_remainder"].__setitem__(0, 999), e.__setitem__("source_input", s)))
+    add("fused_verifier_domain_relabeling", lambda _r, _s, e: e.__setitem__("verifier_domain", "different-domain"))
+    add("fused_statement_version_relabeling", lambda _r, _s, e: e.__setitem__("statement_version", "different-statement"))
+    add("fused_proof_byte_tamper", lambda _r, _s, e: mutate_same_size_fused_proof(e), run_native=True)
+    add("unknown_receipt_key_injection", lambda r, _s, _e: r.__setitem__("unexpected", "claim smuggling"))
+    return cases
+
+
+def validate_mutation_results(mutation_results: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `mutation_results`

(ANN401)